mne/viz/_brain/mplcanvas.py

```diff
@@ -30,16 +31,21 @@ def __init__(self, brain, width, height, dpi):
             pass
         with context:
             self.fig = Figure(figsize=(width, height), dpi=dpi)
-        self.canvas = FigureCanvasQTAgg(self.fig)
-        self.axes = self.fig.add_subplot(111)
+        self.notebook = notebook
+        if self.notebook:
+            self.fig, self.axes = plt.subplots()
```

Comment:
if you set the `figsize` and `dpi` and such it will even be the right size! Not 100% sure how you get it to show up below the brain plot, probably has to do with when each is instantiated which seems like it could be a pain to manage

Reply:
I don't know where to get correct values for `dpi`. We were using Qt for that before

Reply:
I would try just setting it to 100. I don't know if it will matter. If it does we can figure out the right thing to do. One reason you need it is because there `figsize` is in inches, so you need to know the DPI to get from the pixel width to the `figsize`.